src/styles/blocks/product_card/_card.scss

```diff
@@ -0,0 +1,141 @@
+$card-width: 200px;
+$card-height: 408px;
+$other-width: 166px;
+$button-height: 40px;
+$bottom-margin-image: 40px;
+$bottom-margin-name: 4px;
+$bottom-margin-box: 16px;
+$bottom-margin-reviews: 24px;
+$bottom-margin-price: 16px;
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: #{$card-width};
+  height: #{$card-height};
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: rgba(255, 255, 255, 1);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  transition: all 0.3s ease;
```

Comment:
in every place where you use transition you should define every property instead of using all